frontend/sw360-portlet/src/main/webapp/html/projects/view.jsp

```diff
@@ -300,47 +300,81 @@
+                        if (tableData[idx]) {
+	                        pageIds.push(tableData[idx].id);
+
+	                        cell = projectsTable.cell(idx, clearingSummaryColumnIndex);
+	                        cell.data("Loading...");
+                        } else {
+                            break;
+                        }
+                    }
+                    queryPagesPageIds.push(pageIds);
+                }
+                postClearingStateSummaryRequest(queryPagesPageIds);
+            }
+
+            function postClearingStateSummaryRequest(idSets) {
+                if (idSets.length < 1) {
+                    return;
+                }
+
+                var ids = idSets.shift();
+                $.ajax({
+                    type: 'POST',
+                    url: '<%=loadClearingStateAjaxURL%>',
+                    cache: false,
+                    data: {
+                        "<portlet:namespace/><%=Project._Fields.ID%>": ids
+                    },
+                    success: function (response) {
+                        for (var i = 0; i < response.length; i++) {
+                            var cell_clearingsummary = projectsTable.cell("#" + response[i].id, clearingSummaryColumnIndex);
+                            cell_clearingsummary.data(displayClearingStateSummary(response[i].clearing));
+                        }
+                    },
+                    error: function () {
+                        for (var i = 0; i < ids.length; i++) {
+                            var cell = projectsTable.cell("#" + ids[i], clearingSummaryColumnIndex);
+                            cell.data("Failed to load");
+                        }
+                    },
+                    complete: function(xhr, status) {
+                            /* even though former requests might have failed, we want to start all further ones */
+                            postClearingStateSummaryRequest(idSets);
+                    }
+                });
+            }
+
+            function displayClearingStateSummary(clearing){
+                function d(v) { return v == undefined ? 0 : v; }
+
+                var resultElementAsString = "<span class=\"clearingstate\" title=\"Necessary data not found on server!\">Not available</span>",
+                    releaseCount,
+                    approvedCount;
+
+                if (clearing) {
+                    releaseCount = d(clearing.newRelease) + d(clearing.underClearing) + d(clearing.underClearingByProjectTeam) + d(clearing.reportAvailable) + d(clearing.approved);
+                    approvedCount = d(clearing.approved);
+                    resultElementAsString = "<span class=\"clearingstate\" title=\"" + approvedCount + (approvedCount === 1 ? " release" : " releases") + " out of " + releaseCount + (approvedCount === 1 ? " has" : " have") + " been approved (including subprojects)!\">" + approvedCount + "/" + releaseCount + "</span>";
```

Comment:
I don't think this sentence deserves an exclamation mark. Secondly, to lessen the confusion of our users (or discourage misconceptions, if you will), we shouldn't say that releases have been approved, but only that they have approved clearing reports.

Reply:
done in new commit